contracts/Monoplasma.sol

```diff
@@ -54,6 +57,11 @@ contract Monoplasma is BalanceVerifier, Ownable {
         operator = newOperator;
         emit OperatorChanged(newOperator);
     }
+    function setAdminFee(uint24 _adminFee) public onlyOwner {
+        require(adminFee <= 1000000, "Admin fee cannot be greater than 1");
```

Comment:
...then you can also use the "1 ether" form here  (:

Reply:
"ether" didn't work here, I think because of 4.x pragma. Should work in 5.x. 1e18 works